_posts/2017-08-04-thanks-team-my-time-at-nypl.md

```diff
@@ -26,7 +26,7 @@ I have a lot of people to thank for sharing this time with me. 🙏
 
 🎏 **Ho-Ling**, your enthusiasm is endless and endlessly contagious, you are infinitely funny and kind and so generous with snacks. 🔌 **Greg**, I will miss catching your face when you are trying to solve a particularly hard problem, and your pleasant demeanor even when we are both grumpy. 🎲 **Kevin**, I appreciate your board game night initiative and apologize for my partial attendance. 💻 **Jobin**, I will miss your optimism.
 
-💾 **Nick**, when you first started I thought you were a jerk because you are sometimes bad at Twitter, but I was wrong. I now know you are a true ally and valuable asset to the digital preservation community, both within NYPL and globally, and look forward to scheming with you in the future. 📟 **Alex**, you are and always will be the foremost cyberpunk archivist out there. I've already missed you and our bi-weekly archives check-ins, but will miss your genius even more. 💽 **A/V Preservation Unit and Special Collections Unit**, wish we had more time to work together and glad to know many of you socially.
+💾 **Nick**, when you first started I thought you were a jerk because you are often terrible at Twitter, but I was wrong. I now know you are a true ally and valuable asset to the digital preservation community, both within NYPL and globally, and look forward to scheming with you in the future. 📟 **Alex**, you are and always will be the foremost cyberpunk archivist out there. I've already missed you and our bi-weekly archives check-ins, but will miss your genius even more. 💽 **A/V Preservation Unit and Special Collections Unit**, wish we had more time to work together and glad to know many of you socially.
```

Comment:
Disagree, you are not often terrible! I recommend "once said something that people did not like on Twitter"?